ntbs-service/Pages/Shared/_NotificationLayout.cshtml

```diff
@@ -1,54 +1,78 @@
+@model ntbs_service.Pages_Notifications.NotificationModelBase
 @{
     Layout = "../Shared/_Layout.cshtml";
-    var notificationId = Context.Request.Query["id"].ToString();
     var splitUrlPath = Context.Request.Path.ToString().Split("/");
     var currentPage = splitUrlPath[splitUrlPath.Length-1];
 }
 <!DOCTYPE html>
 <html>
 <body>

```

Comment:
Hmmm, is this bit here right, given that we're extending the basic layout anyways? 🤔